module/Entra/AdditionalFunctions/Get-EntraAuditSignInLogs.ps1

```diff
@@ -0,0 +1,118 @@
+function Get-EntraAuditSignInLogs {
+    [CmdletBinding(DefaultParameterSetName = 'GetQuery')]
+    param (
+
+    [Parameter(ParameterSetName = "GetById", ValueFromPipeline = $true, ValueFromPipelineByPropertyName = $true)]
+    [System.String] $ObjectId,
+
+    [Parameter(ParameterSetName = "GetQuery", ValueFromPipeline = $true, ValueFromPipelineByPropertyName = $true)]
+    [System.Nullable`1[System.Int32]] $Top,
```

Comment:
Why is `$Top`  "[System.Nullable`1[System.Int32]]", and not just "[System.Int32]"?